src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="Logo image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item is-active__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item nav__item-margin-left">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item nav__item-margin-left">
+            <a class="nav__link" href="#smarthphones">Smartphones</a>
+          </li>
+          <li class="nav__item nav__item-margin-left">
+            <a class="nav__link"
+                data-qa="hover"
+                href="#latops-computers"
+              >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item nav__item-margin-left">
```

Comment:
You don't need additional classes for margin, use css pseudo classes like :nth-child

Reply:
I changed added `nth-child`.